dashboards/f5/f5-big-ip.json

```diff
@@ -0,0 +1,1184 @@
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "units": {
+                "unit": "BITS_PER_SECOND"
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Top 10 Transmit bps",
+            "layout": {
+              "column": 7,
+              "row": 9,
+              "width": 6,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "FROM F5BigIpPoolSample SELECT average(pool.inDataInBytesPerSecond) AS 'Rx bps' WHERE reportingEndpoint IN ( {{f5_system}} ) AND displayName IN ( {{f5_pool}} ) FACET reportingEndpoint AS 'System', displayName AS 'Node' TIMESERIES LIMIT 10"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "units": {
+                "unit": "BITS_PER_SECOND"
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          }
+        ]
+      },
+      {
+        "name": "Pool Members",
+        "description": null,
+        "widgets": [
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Pool Members\nA [pool member](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/6.html#conceptid) is a logical object that represents a physical node (server), on the network. You then associate the pool with a virtual server on the BIG-IP® system. Once you have assigned a pool to a virtual server, Local Traffic Manager (LTM®) directs traffic coming into the virtual server to a member of that pool. An individual pool member can belong to one or multiple pools, depending on how you want to manage your network traffic."
```

Comment:
Lets update this url:
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg